layouts/partials/nav.html

```diff
@@ -28,18 +28,14 @@
   </ul>
   <ul class="invisible lg:visible float-right h-16 flex items-center">
     <li class="text-right inline mr-6"><a href='{{ "/#applications" | relURL }}' class="text-Grey-600 hover:no-underline">Applications</a></li>
-    <li class="text-right inline mr-6"><a href='{{ "/how-to-design-a-verifiable-system/" | relURL }}' class="text-Grey-600 hover:no-underline">Verifiable Design</a></li>
-    <li class="text-right inline mr-6"><a href='{{ "/#trillian" | relURL }}' class="text-Grey-600 hover:no-underline">Trillian</a></li>
-    <li class="text-right inline mr-6"><a href='{{ "/verifiable-data-structures/" | relURL }}' class="text-Grey-600 hover:no-underline">Verifiable Data Structures</a></li>
+    <li class="text-right inline mr-6"><a href='{{ "/summit/" | relURL }}' class="text-Grey-600 hover:no-underline">Transparency.dev Summit</a></li>
     <li class="text-right inline mr-6"><a href='{{ "/articles/" | relURL }}' class="text-Grey-600 hover:no-underline">Articles</a></li>
     <li class="text-right inline mr-6"><a href="https://blog.transparency.dev" class="text-Grey-600 hover:no-underline">Blog</a></li>
   </ul>
 </nav>
 <ul id="mobileMenu" class="hidden bg-white p-8 fixed w-full z-50">
   <li class="gmb-2"><a href='{{ "/#applications" | relURL }}' class="text-Grey-600 hover:no-underline">Applications</a></li>
-  <li class="gmb-2"><a href='{{ "/how-to-design-a-verifiable-system/" | relURL }}' class="text-Grey-600 hover:no-underline">Verifiable Design</a></li>
-  <li class="gmb-2"><a href='{{ "/#trillian" | relURL }}' class="text-Grey-600 hover:no-underline">Trillian</a></li>
-  <li class="gmb-2"><a href='{{ "/verifiable-data-structures/" | relURL }}' class="text-Grey-600 hover:no-underline">Verifiable Data Structures</a></li>
+  <li class="gmb-2"><a href='{{ "/summit/" | relURL }}' class="text-Grey-600 hover:no-underline">Transparency.dev Summit</a></li>
```

Comment:
Nit: should this be `2025 Summit` rather to make it shorter and more visible that hit's happening soon?

Reply:
Yeah was torn about that... I didn't want it to go out of date but now you've commented this I'll update it (plus it will match the blog too!)